app/models/event.rb

```diff
@@ -4,4 +4,13 @@ class Event < ActiveRecord::Base
   validates :slug, presence: true, uniqueness:true
   validates :location, presence: true, length: { maximum: 128 }
   validates :date, presence: true
+
+  def self.upcoming
+    where('date > ?', DateTime.now).order(date: :desc)
+  end
+
+  def self.previous
```

Comment:
This stuff will probably change anyway when pagination is added. I'll change it if needs be when we do that.